simpleviewinc/keg-herkin
# Update file/folder names to be correct casing

## Updates
* Fix file names that were using the incorrect casing, given imports
* The docker environment in which the herkin tap runs is case-sensitive, so these inconsistencies were a problem
* Also updated my local git config to be case-sensitive, which others should do as well: `keg herkin && git config core.ignorecase false`)

## Testing
* Switch to this branch
* Run `keg herkin start` in a directory w/ a `herkin.config` file
* Verify the tap starts
